File: Daily Pratice Question/JAVASCRIPT/question(7).js
// 7. Write a function that simulates a traffic light system.
//  The function should take the current light color (red, yellow, green) as
//   input and print the corresponding action:
// "red" → "Stop"
// "yellow" → "Slow down"
// "green" → "Go"

var light="yellow"

function trafficLight(){
    if(light=="red"){
        console.log("stop")
    }
    else if(light=="yellow"){
        console.log("slow down")
    }
    else if(light=="green"){
        console.log("go")
    }
}
trafficLight()

File: Daily Pratice Question/C LANGUAGE/Question(7).c
// 7. Write a function that simulates a traffic light system. 
// The function should take the current light color (red, yellow, green) as 
// input and print the corresponding action:
// "red" → "Stop"
// "yellow" → "Slow down"
// "green" → "Go"



#include <stdio.h>
#include <string.h>
void trafficLight(char*light){
    if(strcmp(light,"red")==0){
        printf("Stop");
    }
    else if(strcmp(light,"yellow")==0){
        printf("slow down");
    }else if(strcmp(light,"green")==0){
        printf("Go");
    }
}
int main() {
    char light[]="yellow";
    trafficLight(light);
}